x-pack/filebeat/module/panw/panos/config/input.yml

```diff
@@ -51,6 +51,8 @@ processors:
       omit_empty: true
       fail_on_error: false
       mappings:
+        panw.panos.type: 3
```

Comment:
It looks like type=3 and subtype=4 are common across all of the log types in PAN-OS. So maybe the pipeline should set them for all log types by placing this around line 38?